Changed deprecated method call for FluentWait ()

boris779 opened the pull request:
Added new method calls for withTimeout and pollingEvery, with the usage of Duration.ofSeconds()

boris779 committed:
Changed deprecated method call for FluentWait (


Added new method calls for withTimeout and pollingEvery, with the usage of Duration.ofSeconds()

diemol reviewed (changes requested):
Can you please add this to the other translated languages?

boris779 committed:
changed FluenWait Call

changed method call for withTimeout and pollinEvery due to deprecated method call

boris779 commented:
Done

diemol reviewed (approved):
Thanks @boris779!
